package.json

```diff
@@ -33,23 +34,32 @@
   "homepage": "https://github.com/sglanzer/ember-cli-blanket",
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.2",
+    "chai": "^2.2.0",
     "ember-cli": "0.2.2",
-	"ember-cli-htmlbars": "0.7.4",
+    "ember-cli-app-version": "0.3.3",
     "ember-cli-content-security-policy": "0.4.0",
     "ember-cli-dependency-checker": "0.0.8",
+    "ember-cli-htmlbars": "0.7.4",
     "ember-cli-ic-ajax": "0.1.1",
     "ember-cli-inject-live-reload": "^1.3.0",
     "ember-cli-qunit": "0.3.9",
-    "ember-cli-app-version": "0.3.3",
     "ember-cli-uglify": "1.0.1",
-    "ember-export-application-global": "^1.0.2"
+    "ember-export-application-global": "^1.0.2",
+    "glob": "^5.0.3",
+    "mocha": "^2.2.1",
+    "tmp-sync": "^1.0.1"
   },
   "dependencies": {
+    "body-parser": "^1.12.2",
     "broccoli-funnel": "^0.2.2",
-    "broccoli-merge-trees": "^0.2.1"
+    "broccoli-merge-trees": "^0.2.1",
+    "fs-extra": "^0.18.0",
+    "core-object": "^1.1.0",
+    "lodash": "^3.6.0"
   },
   "ember-addon": {
     "after": "ember-cli-qunit",
```

Comment:
Fair question on the after hook. I'll give it a shot when I take a final pass on this

Plan is to squash (will be Monday before I can do this)